src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -1,16 +1,22 @@
 #pragma once
 
+#include <google/protobuf/repeated_field.h>
+
 #include "software/backend/robot_status.h"
 #include "software/multithreading/subject.h"
 #include "software/multithreading/threaded_observer.h"
+#include "software/proto/sensor_msg.pb.h"
 #include "software/sensor_fusion/filter/ball_filter.h"
 #include "software/sensor_fusion/filter/robot_team_filter.h"
 #include "software/sensor_fusion/refbox_data.h"
+#include "software/sensor_fusion/ssl_protobuf_reader.h"
 #include "software/sensor_fusion/vision_detection.h"
 #include "software/world/ball.h"
 #include "software/world/team.h"
 #include "software/world/world.h"
 
+using namespace google::protobuf;
```

Comment:
I'm a big fan of *not* using 'using', especially because this is a header.

Can we change this to using google::protobuf as proto:: or something similar?

Reply:
oh yea it's generally good practice to avoid `using` namespaces in a header because anything that includes the header will also `use` that namespace. I'd prefer to move it to the `cpp` file and just use the full namespace names in the header.

@garethellis0

Reply:
It's used in `RepeatedPtrField<TbotsRobotMsg> tbots_robot_msgs` which is defined in the header. I generally prefer not using `namespaces` too, but this one is a bit verbose.